src/main/java/org/terasology/gooeyDefence/PathfindingSystem.java

```diff
@@ -75,20 +81,48 @@ public void onChangedBlock(OnChangedBlock event, EntityRef entity) {
         }
     }
 
+    /**
+     * Called to request an enemy be re-pathed.
+     *
+     * @see RepathEnemyRequest
+     */
+    @ReceiveEvent
+    public void onRepathEnemyRequest(RepathEnemyRequest event, EntityRef entity, LocationComponent locationComponent) {
+        /* Pause the enemy */
```

Comment:
can extract the below two lines into a function pauseEnemy and remove the comment